pkg/polaris/patcher.go

```diff
@@ -0,0 +1,192 @@
+	content = strings.ReplaceAll(content, "${ORG_NAME}", polaris.OrganizationDetails.OrganizationProvisionOrganizationName)
+	content = strings.ReplaceAll(content, "${ADMIN_NAME}", polaris.OrganizationDetails.OrganizationProvisionAdminName)
+	content = strings.ReplaceAll(content, "${ADMIN_USERNAME}", polaris.OrganizationDetails.OrganizationProvisionAdminUsername)
+	content = strings.ReplaceAll(content, "${ADMIN_EMAIL}", polaris.OrganizationDetails.OrganizationProvisionAdminEmail)
+	content = strings.ReplaceAll(content, "${SEAT_COUNT}", polaris.OrganizationDetails.OrganizationProvisionLicenseSeatCount)
+	content = strings.ReplaceAll(content, "${TYPE}", polaris.OrganizationDetails.OrganizationProvisionLicenseType)
+	content = strings.ReplaceAll(content, "${RESULTS_START_DATE}", polaris.OrganizationDetails.OrganizationProvisionResultsStartDate)
+	content = strings.ReplaceAll(content, "${RESULTS_END_DATE}", polaris.OrganizationDetails.OrganizationProvisionResultsEndDate)
+	content = strings.ReplaceAll(content, "${RETENTION_START_DATE}", polaris.OrganizationDetails.OrganizationProvisionRetentionStartDate)
+	content = strings.ReplaceAll(content, "${RETENTION_END_DATE}", polaris.OrganizationDetails.OrganizationProvisionRetentionEndDate)
+
+	// TODO https://github.com/blackducksoftware/synopsys-operator/issues/943
+	if len(polaris.Repository) != 0 {
+		content = strings.ReplaceAll(content, "gcr.io/snps-swip-staging", polaris.Repository)
+	}
+
+	mapOfUniqueIDToBaseRuntimeObject := ConvertYamlFileToRuntimeObjects(content)
+	mapOfUniqueIDToBaseRuntimeObject = removeTestManifests(mapOfUniqueIDToBaseRuntimeObject)
+
+	patcher := Patcher{
+		polaris:                          polaris,
+		mapOfUniqueIDToBaseRuntimeObject: mapOfUniqueIDToBaseRuntimeObject,
+	}
+
+	return patcher.patch()
+}
+
+type Patcher struct {
+	polaris                          Polaris
+	mapOfUniqueIDToBaseRuntimeObject map[string]runtime.Object
+}
+
+func (p *Patcher) patch() (map[string]runtime.Object, error) {
+	patches := []func() error{
+		p.patchNamespace,
+		p.patchVersionLabel,
+	}
+	for _, f := range patches {
+		err := f()
+		if err != nil {
+			return nil, err
+		}
+	}
+	return p.mapOfUniqueIDToBaseRuntimeObject, nil
+}
+
+func (p *Patcher) patchNamespace() error {
+	accessor := meta.NewAccessor()
+	for _, runtimeObject := range p.mapOfUniqueIDToBaseRuntimeObject {
+		accessor.SetNamespace(runtimeObject, p.polaris.Namespace)
+	}
+	return nil
+}
+
+func (p *Patcher) patchVersionLabel() error {
+	accessor := meta.NewAccessor()
+	for _, runtimeObject := range p.mapOfUniqueIDToBaseRuntimeObject {
+		labels, err := accessor.Labels(runtimeObject)
+		if err != nil {
+			return err
+		}
+
+		if labels == nil {
+			labels = make(map[string]string)
+		}
+
+		labels["polaris.synopsys.com/version"] = p.polaris.Version
+		labels["polaris.synopsys.com/environment"] = p.polaris.Namespace
+
+		if err := accessor.SetLabels(runtimeObject, labels); err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
same comment as for patchNamespace?... isn't the whole idea, now you only need placeholders?

Reply:
it has been removed. Those labels are not needed anymore